src/common/rbac.ts

```diff
@@ -10,6 +10,16 @@ export type KubeResource =
   "endpoints" | "customresourcedefinitions" | "horizontalpodautoscalers" | "podsecuritypolicies" | "poddisruptionbudgets" |
   "roles" | "clusterroles" | "rolebindings" | "clusterrolebindings" | "serviceaccounts";
 
+export enum KubeResourceEnum {
+  pods = "pods",
+  deployments = "deployments",
+  cronJobs = "cronjobs",
+  jobs = "jobs",
+  replicasets = "replicasets",
+  daemonsets = "daemonsets",
+  statefulsets = "statefulsets",
+}
```

Comment:
At the moment yes. Union type above doesn't do the same. 

The real solution would be to make `KubeResource`s comply to Open Closed Principle. But that's out of scope here, but when done, we can safely remove whole file and all contents from it.

Reply:
What do you mean by "do the same"? And it is impossible for a type to be OCP in TS so that point is moot.

Reply:
Here we have a list of magic strings for entities which also relate to functionalities which are spread-and-lost around the codebase, making it not OCP. If what it means to be an entity of "KubeResource" was specified in a centralized place, it would be OCP.

I propose not investing a lot of effort in this before OCP happens.

Reply:
Then just make it `string`? I don't see the point of adding this enum.

Reply:
Done.